js/request.js

```diff
@@ -150,7 +150,6 @@ exports.request = function(req, res, opts, handler) {
       var terminationMessage = 'Terminating server at http://[localhost]:' + opts.port + ' for ' + opts['module-path'] + ' / ' + opts.handler;
       res.end(terminationMessage + '\n');
       console.info(terminationMessage);
-      server.close();
```

Comment:
I think the problem it was not using strict mode (or ES6?) and my code path is not issuing a DELETE (as it works for me!). See stack trace below:
```
/Users/ed/BBC/src/pcs-fbd-bling/bling-test/cucumber/.gems/ruby/2.2.0/gems/aws-lambda-runner-1.4.4/js/request.js:153
      server.close();
      ^

ReferenceError: server is not defined
    at /Users/ed/BBC/src/pcs-fbd-bling/bling-test/cucumber/.gems/ruby/2.2.0/gems/aws-lambda-runner-1.4.4/js/request.js:153:7
    at Object.exports.request (/Users/ed/BBC/src/pcs-fbd-bling/bling-test/cucumber/.gems/ruby/2.2.0/gems/aws-lambda-runner-1.4.4/js/request.js:154:7)
    at Server.<anonymous> (/Users/ed/BBC/src/pcs-fbd-bling/bling-test/cucumber/.gems/ruby/2.2.0/gems/aws-lambda-runner-1.4.4/js/startup.js:17:11)
    at emitTwo (events.js:106:13)
    at Server.emit (events.js:191:7)
    at HTTPParser.parserOnIncoming [as onIncoming] (_http_server.js:543:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:105:23)

```